package.json

```diff
@@ -85,11 +85,12 @@
     "jest-coverage": "yarn jest --coverage",
     "lint": "eslint . --cache --ext js,md",
     "lint-es5-build": "eslint --no-eslintrc --no-ignore --env=browser packages/*/build-es5",
-    "lint:md": "prettier docs/**/*.md packages/*/README.md packages/*/docs/**/*.md website/blog/*.md .github/*.md CHANGELOG.md CONTRIBUTING.md website/README.md README.md --write",
+    "lint:md": "prettier '**/*.md' --write",
```

Comment:
I confirmed that these two commands are equivalent, not sure what we were filtering before

Reply:
We were supposed to not edit README.md, seems like that changed

Reply:
🤔README.md is the last file listed in the original command